.github/workflows/main.yml

```diff
@@ -89,3 +87,24 @@ jobs:
         bazel --batch build \
           --keep_going \
           "//:ray_pkg"  # TODO(mehrdadn): Should be "//:*", but we get a linking error with _streaming.so
+  jenkins:
+    name: docker integration
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        test-flag:
+          - "--run-tune"
+          - "--run-doc"
+          - "--run-sgd"
+      fail-fast: false
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - name: Run Jenkins Tests ${{matrix.test-flag}}
+        continue-on-error: false
+        shell: bash
```

Comment:
When you merge in master, don't forget to change this to be like the lines in there, which I think should be:

    shell: bash -e -o pipefail -l {0}